src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(String color, int radius) {
```

Comment:
The type of `radius` parameter in the constructor should be `double` instead of `int`. Change the parameter type to `double` for correct calculations. 🧮